config/serverless.yml

```
@@ -1,3 +1,9 @@
+# Common serverless config
+
+## By default, we exclude all solutions' plugins in serverless
+## We let each solution override this by explicitly including their own paths
+plugins.excludedPluginGroups: ['observability', 'security', 'search', 'chat']
```

WDYT about sticking with the allow list approach?

```suggestion
plugins.includedPluginGroups: []
```

---

If we change the `includedPluginGroups` to not default to `[]` we can treat `undefined` as having a different meaning and might not need the `plugins.excludedPluginGroups` config. So the filtering logic would be smth like:

`config.includedPluginGroups ? <apply-filtering> : <do-not-filter-aka-allow-all>`

---

Nevermind, it's better if core does not rely on serverless (or cloud) configuration for that.
I created an `allowlistPluginGroups` as we planned. This will be more flexible, and perhaps reusable for 'cloud' deployments.